src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -319,7 +319,61 @@
   end
 
   describe '#user_and_role' do
-    skip('Please add some tests')
+    let(:user) { create(:user) }
+    let(:nobody) { create(:user_nobody) }
```

Comment:
I'm not talking about the factory, I'm talking about the whole ``#user_and_role`` specs. They have nothing todo with 
4bc3e992e2f66dc96a910df924f7a98015514573 (Refactor setup_codemirror_editor for webui helper), do they?